src/test/java/catascore/KeyGeneratorImplTest.java

```diff
@@ -0,0 +1,54 @@
+package catascore;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+public class KeyGeneratorImplTest {
+
+    final static int N_CHAR = 6;
+
+    @Test
+    public void generateSixCharacterKey() throws Exception{
+        KeyGenerator keyGenerator = new KeyGeneratorImpl();
+        assertEquals(N_CHAR, keyGenerator.generateKey().length());
+    }
+
+    @Test
+    public void generateUppercaseKey() throws Exception{
+        KeyGenerator keyGenerator = new KeyGeneratorImpl();
+        String key = null;
+        key = keyGenerator.generateKey();
+        assertEquals(key.toUpperCase(),key);
+    }
+
+    @Test
+    public void  generateUniqueKey() throws Exception{
+        KeyGenerator keyGenerator = new KeyGeneratorImpl();
+        assertNotEquals(keyGenerator.generateKey(),keyGenerator.generateKey());
```

Comment:
I think it's not clear enough that you call the same method, It will be good that you can differentiate each Key, the old and the new.

Reply:
Changed, thanks